config/account.go

```diff
@@ -81,6 +94,34 @@ func (pf *AccountPriceFloors) validate(errs []error) []error {
 		errs = append(errs, fmt.Errorf(`account_defaults.price_floors.max_schema_dims should be between 0 and 20`))
 	}
 
+	if pf.Fetcher.Period > pf.Fetcher.MaxAge {
+		errs = append(errs, fmt.Errorf(`account_defaults.price_floors.fetch.period_sec should be less than account_defaults.price_floors.fetch.max_age_sec`))
+	}
+
+	if pf.Fetcher.Period < 300 {
+		errs = append(errs, fmt.Errorf(`account_defaults.price_floors.fetch.period_sec should not be less than 300 seconds`))
+	}
+
+	if !(pf.Fetcher.MaxAge >= 600 && pf.Fetcher.MaxAge < math.MaxInt32) {
+		errs = append(errs, fmt.Errorf(`account_defaults.price_floors.fetch.max_age_sec should not be less than 600 seconds and greater than maximum integer value`))
+	}
+
+	if !(pf.Fetcher.Timeout > 10 && pf.Fetcher.Timeout < 10000) {
+		errs = append(errs, fmt.Errorf(`account_defaults.price_floors.fetch.timeout_ms should be between 10 to 10,000 miliseconds`))
+	}
+
+	if !(pf.Fetcher.MaxRules >= 0 && pf.Fetcher.MaxRules < math.MaxInt32) {
```

Comment:
It is not necessary to check `pf.Fetcher.MaxRules < math.MaxInt32` since `MaxRules` (and `MaxFileSize`) are 32 bit integers, this expression must always be true.

Reply:
Done